### ISSUE TITLE
Second digit not cleared for time column in interactive mode. So align time value to right

### ISSUE THREAD

**Issue by darkblaze69** [flightaware/dump1090#174]
Second digit not cleared on updates for time column in interactive mode.
Apparently needs aligning time value to right. So format should be like 
`"%s%06X %-4s  %-4s  %-8s %6s %3s  %3s  %7s %8s %5.1f %5d %3.0f"`
instead of current 
`"%s%06X %-4s  %-4s  %-8s %6s %3s  %3s  %7s %8s %5.1f %5d %*.0f"`

**Comment by mutability** [comment]
Yeah, this is a regression from when a PR to add an option to support longer timeouts was merged.

You can't just change the format string quite like that though.

**Comment by darkblaze69** [comment]
The problem is still not fixed. What's the proper way to fix this then?
